test/src/device-request-prompt.spec.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2023 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import expect from 'expect';
+import {TimeoutError} from 'puppeteer';
+
+import {launch} from './mocha-utils.js';
+
+describe('device request prompt', function () {
+  let state: Awaited<ReturnType<typeof launch>>;
+
+  before(async () => {
+    state = await launch(
+      {
+        args: ['--enable-features=WebBluetoothNewPermissionsBackend'],
+        ignoreHTTPSErrors: true,
```

Comment:
I believe we don't need this option.

Reply:
Without this option:

` Error: net::ERR_CERT_AUTHORITY_INVALID at https://localhost:63413/empty.html`

Reply:
Unless you want me to use `server` instead of `httpsServer`?

Reply:
Let's leave it as is then, I didn't manage to verify it but its possible that this feature is enabled only for secured sites (https).